src/pages/index.js

```diff
@@ -61,14 +95,32 @@ const editFormValidator = new FormValidator(
   validationSettings
 );
 const addFormValidator = new FormValidator(addCardForm, validationSettings);
+const avatarFormValidator = new FormValidator(
+  avatarEditForm,
+  validationSettings
+);
 addFormValidator.enableValidation();
 editFormValidator.enableValidation();
+avatarFormValidator.enableValidation();
 
 const handleFormSubmit = (formData) => {
```

Comment:
`handleFormSubmit`

There are 3 modals with different buttons and forms in the project. You need to **name** variables and functions the way all could understand what they are.


`handleCardFormSubmit`